index.bs

```diff
@@ -4675,7 +4675,8 @@ To <dfn>queue reports for delivery</dfn> given a [=set=] of
         Note: In order to support sending, waiting, and retries across various
          forms of interruption, including shutdown, the user agent may need to
          persist reports that are in the process of being sent in some other
-         storage.
+         storage. These persisted reports should have a lifetime of no more than
```

Comment:
It might also be beneficial to delete without sending any reports older than this in the https://wicg.github.io/attribution-reporting-api/#queue-reports-for-delivery algorithm.